src/components/EditSections/EditUserInfo/components/ExternalLinkModal/ExternalLinkModal.js

```diff
@@ -23,26 +23,37 @@ const ExternalLinkModal = ({
   const [inputValue, setInputValue] = useState('');
   const previousInputValue = useRef(profilePictureLink);
   const [disabled, setDisabled] = useState(false);
-  const [error, setError] = useState(false);
-  const externalURLValidityError = error ?
-    <FormattedMessage id="ui-users.information.profilePicture.externalLink.modal.externalURL.errorMessage" />
-    : null;
+  const [externalURLValidityError, setExternalURLValidityError] = useState(null);
 
   useEffect(() => {
     setInputValue(profilePictureLink);
-    setError(false);
   }, [profilePictureLink]);
 
   useEffect(() => {
+    setExternalURLValidityError(null);
     if (inputValue) {
       setDisabled(previousInputValue.current === inputValue);
-      setError(false);
     } else {
       setDisabled(true);
     }
   }, [inputValue]);
 
-  const handleSave = () => {
+  const handleSave = async () => {
+    setExternalURLValidityError(null);
+    if (!inputValue) return;
```

Comment:
This code is never hit as when there is no input, save button is disabled.

Reply:
yes, it is not needed now, removed.